go/master/client.go

```diff
@@ -112,6 +112,11 @@ func (c *Client) taskFinished(taskID int) error {
 	return c.conn.Call("Service.TaskFinished", taskID, nil)
 }
 
+// TaskFailed tell the master server as task is failed.
+func (c *Client) taskFailed(taskID TaskID) error {
+	return c.conn.Call("Service.TaskFinished", taskID, nil)
```

Comment:
@typhoonzero No worries, that's why we have multiple developers reviewing, we make mistake :)